lib/index.ts

```diff
@@ -318,13 +320,7 @@ export class Worker {
 					patch,
 				);
 			},
-			enqueueAction: (...args) => {
-				return this.enqueueAction(...args);
-			},
-			cards: {
-				...CORE_CARDS,
-				...self.getTypeContracts(),
-			},
+			enqueueAction: this.enqueueAction,
```

Comment:
Does `this` need to be bound for the assignment to work correctly? 🤔

Reply:
This change was reverted @LucianBuzzo.